locales/ja.json

```diff
@@ -1,23 +1,23 @@
 {
-  "keyword-meta": "freeCodeCamp, programming, front-end, programmer, article, regular expressions, Python, JavaScript, AWS, JSON, HTML, CSS, Bootstrap, React, Vue, Webpack",
-  "banner": "Learn to code — <span>free 3,000-hour curriculum</span>",
-  "learn-to-code-cta": "Learn to code for free. freeCodeCamp's open source curriculum has helped more than 40,000 people get jobs as developers.",
-  "learn-to-code-cta-btn": "Get started",
-  "footer-desc-1": "freeCodeCamp is a donor-supported tax-exempt 501(c)(3) nonprofit organization (United States Federal Tax Identification Number: 82-0779546)",
-  "footer-desc-2": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
-  "footer-desc-3": "Donations to freeCodeCamp go toward our education initiatives and help pay for servers, services, and staff.",
-  "footer-donation": "You can <a id='footer-donation' class='inline' rel='noopener noreferrer' target='_blank'>make a tax-deductible donation here</a>.",
-  "footer-trending-guides-header": "Trending Guides",
-  "footer-our-nonprofit-header": "Our Nonprofit",
-  "footer-about": "About",
-  "footer-alumni": "Alumni Network",
-  "footer-open-source": "Open Source",
-  "footer-shop": "Shop",
-  "footer-support": "Support",
-  "footer-sponsors": "Sponsors",
-  "footer-honesty": "Academic Honesty",
-  "footer-coc": "Code of Conduct",
-  "footer-privacy": "Privacy Policy",
-  "footer-tos": "Terms of Service",
-  "footer-copyright": "Copyright Policy"
+  "keyword-meta": "freeCodeCamp, フリーコードキャンプ, プログラミング, フロントエンド, プログラマー, 記事, 正規表現, Python, JavaScript, AWS, JSON, HTML, CSS, Bootstrap, React, Vue, Webpack",
+  "banner": "コーディングを学ぶ — <span>約 3,000 時間のカリキュラムが無料</span>",
```

Comment:
You can set this to link to what you want, with matching translation. The link is the `banner-url` link in the other file. For most languages we set it to link to the chatroom in the specific language.

Reply:
Thanks for letting me know this. But I think people will be more interested in free coding curriculums even if it's not in Japanese yet. So I'm thinking to leave it as is.